composed_api.rst

```diff
@@ -457,14 +755,12 @@ Example
 
 ::
 
-    curl -u <username> http://gridte.rsgis.erdc.dren.mil/te_ba/api/v1/geoname/?geom=POLYGON ((30 10, 40 40, 20 40, 10 20, 30 10))&source=grid
+    curl -u <username> http://gridte.rsgis.erdc.dren.mil/te_ba/api/v2/geoname/?geom=POLYGON ((30 10, 40 40, 20 40, 10 20, 30 10))&source=grid
 
 ::
 
     {
-        "GRiD API": {
-            "API Version": "v1"
-        }, 
+        "API Version": "v2", 
         "name": "Great Sand Sea", 
         "provided_geometry": "POLYGON ((30 10, 40 40, 20 40, 10 20, 30 10))"
```

Comment:
In most other cases, the key would not change from `geom` in the request to `provided_geometry` in the response (although we do change `geom` to `clip_geometry` in one of the other calls). If this will always just be echoing, but not changing, the requested value, it seems we should use the same key.